hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientScheduledFutureProxy.java

```diff
@@ -87,14 +87,16 @@ public ScheduledTaskStatistics getStats() {
             ScheduledExecutorGetStatsFromAddressCodec.ResponseParameters responseParameters =
                     ScheduledExecutorGetStatsFromAddressCodec.decodeResponse(response);
             return new ScheduledTaskStatisticsImpl(responseParameters.totalRuns, responseParameters.lastIdleTimeNanos,
-                    responseParameters.totalRunTimeNanos, responseParameters.totalIdleTimeNanos);
+                    responseParameters.totalRunTimeNanos, responseParameters.totalIdleTimeNanos,
+                    responseParameters.lastRunDurationNanos);
```

Comment:
When server is old this value will not be available. 
You can check if server send the information via
responseParameters.lastRunDurationNanosExists field and handle the missing case.

Reply:
I assumed it would be default val, 0, so I left it like that. Wrong assumption?

Reply:
It is 0 by default.